app/api/waitlists/route.ts

```diff
@@ -38,20 +28,15 @@ export const POST = async (req: NextRequest) => {
   const notEligibleImage: File | null = body.get("files[2]") as unknown as File;
   const errorImage: File | null = body.get("files[3]") as unknown as File;
 
-  if (
-    !name ||
-    !endDate ||
-    !externalUrl ||
-    !address ||
-    !landingImage ||
-    !successImage ||
-    !notEligibleImage ||
-    !errorImage
-  ) {
-    return NextResponse.json(
-      { success: false, message: "Missing required fields" },
-      { status: 400 }
-    );
+  // Prisma call to check if the user has more than 1 waitlist
+  const waitlists = await getUserWaitlists(address!);
```

Comment:
Instead of "getUserWaitlists" we should use a specific function that returns only the count, because getUserWaitlists is more heavy as a computation as it includes a lot of useless data that we don't need here